modules/dnn/test/test_model.cpp

```diff
@@ -806,6 +807,85 @@ TEST_P(Test_Model, TextDetectionByEAST)
     );
 }
 
+TEST_P(Test_Model, YOLOv7)
+{
+    std::string weightPath = _tf("onnx/models/yolov7_not_simplified.onnx");
+    std::string imgPath = _tf("dog_orig_size.jpg");
```

Comment:
JPEG or other loss image formats should not be used in tests as input data.

I have at least 3 configurations with different CRC64 values. Which one has gold values?

Reply:
Now use png instead.